homework_02/_12_Martin_Vrachev/templates/departure_board.html

```diff
@@ -0,0 +1,57 @@
+<style>
+
+table, th, td {
+  text-align: center;
+}
+
+h1 {
+    text-align: center;
+}
+</style>
+
+
+<h1>NORTH STATION INFORMATION</h1>
+
+<div style="width: 100%; font-size: 25px; font-weight: bold;">
+    <div style="width: 50%; float: left; text-align: left;"> {{ weekday }} </div>
+    <div style="width: 50%; float: left; text-align: right;"> Current time </div>
+</div>
+
+<div style="width: 100%; font-size: 25px; font-weight: bold;">
+    <div style="width: 50%; float: left; text-align: left;"> {{ date }} </div>
+    <div style="width: 50%; float: left; text-align: right;"> {{ time }} </div>
+</div>
+
+<div style="margin-top:200px;">
+    <table style="width: 100%;">
+        <tr>
+            <th>Estimated Arrival Time</th>
+            <th>Estimated Departure Time</th>
+            <th>Status</th>
+            <th>Train#</th>
+            <th> Destination </th>
+        </tr>
+        {% for r in response %}
+        <tr>
+            <td>{{ r.attributes.arrival_time }}</td>
+            <td>{{ r.attributes.departure_time }}</td>
```

Comment:
Yo cant directly take departure time because it some times is None you cant take it from schedule